reports/behaviour.py

```diff
@@ -118,6 +118,8 @@ def get_network_activity(dns_lookups=[], http_conv=[], ids_alerts=[], parent_sec
 
             # DNS Lookup Table
             for record in dns_lookups:
+                if not record.get('hostname'):
+                    continue
                 domain = record['hostname']
```

Comment:
This looks strange, if the record could not have the key 'hostname' it should be `if not 'hostname' in record:`. If the key could not be there or the value could be none I think `domain = record.get('hostname')` and `if not domain: continue` looks more natural.